### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by JuanLPalacios**
initialize the project and add components placeholders.

### CODE REVIEW

File: src/Books.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import Book from './components/Book';
+import BookInput from './components/BookInput';
+
+export default function Books() {
+  // eslint-disable-next-line no-unused-vars
```
Comment (bimbolabuari) [inline src/Books.js:6]: Kindly remove all disabled linter for professional and efficient code

Please check for other occurrences 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @JuanLPalacios, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Add all project requirement

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Books.js (hunk 1 above)
Comment (JuanLPalacios) [reply src/Books.js:6]: the setList method is yet to be used but is Unduundubly essential as part of the component hence the comment

**Review by bimbolabuari — APPROVED** [review]
Hi @JuanLPalacios, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**